docs/reference/kubectl/docker-cli-to-kubectl.md

```diff
@@ -26,44 +26,43 @@ CONTAINER ID        IMAGE               COMMAND                  CREATED
 55c103fa1296        nginx               "nginx -g 'daemon of…"   9 seconds ago       Up 9 seconds        0.0.0.0:80->80/tcp   nginx-app
 ```
 
-With kubectl:
+kubectl:
 
 ```shell
 # start the pod running nginx
 $ kubectl run --image=nginx nginx-app --port=80 --env="DOMAIN=cluster"
 deployment "nginx-app" created
 ```
 
-`kubectl run` creates a Deployment named "nginx-app" on Kubernetes cluster >= v1.2. If you are running older versions, it creates replication controllers instead.
-If you want to obtain the old behavior, use `--generator=run/v1` to create replication controllers. See [`kubectl run`](/docs/user-guide/kubectl/{{page.version}}/#run) for more details.
-Note that `kubectl` commands will print the type and name of the resource created or mutated, which can then be used in subsequent commands. Now, we can expose a new Service with the deployment created above:
+On a Kubernetes cluster version 1.2,`kubectl run` command creates a Deployment named "nginx-app". If you are running older versions of Kubernetes cluster, then `kubectl run` command creates replication controllers.
+On a Kubernetes cluster version 1.2, use `--generator=run/v1` to create replication controllers. See [`kubectl run`](/docs/user-guide/kubectl/{{page.version}}/#run) for more information.
```

Comment:
The above two lines can be removed because v1.2 is too old and info contained here are not applicable today.